### PR TITLE
[ADD] sale_order_product_recommendation_quick_add

### PR DESCRIPTION
**Pull request by santostelmo**
This module adds a button on sale order to quick add recommended products. 
User can quick add recommended products in one click without going through the wizard implemented in `sale_order_product_recommendation`.

### CODE REVIEW

**Review by Ricardoalso — APPROVED** [review]
LGTM

File: sale_order_product_recommendation_quick_add/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sale Order Product Recommendation Quick Add",
+    "summary": "Add recommended products to sale order in a single click",
+    "version": "16.0.1.2.1",
```
Comment (simahawk) [inline sale_order_product_recommendation_quick_add/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by simahawk — APPROVED** [review]
LG, version to fix. Pre-approving.

**Pull request merged**